.agents/structure.md

```
@@ -92,6 +92,30 @@ packages/core/src/
   types.ts            # VNodeClass, VNodeProperties, PartialPick
 ```
 
+## List Package Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)